const Exception = () => {
  return <></>;
};

export default Exception;
